.gitignore

```diff
@@ -0,0 +1,10 @@
+### Sass ###
+.sass-cache/
+.scss-cache/
+*.css.map
+*.sass.map
+*.scss.map
+*.css
+
+## System ##
+.DS_Store
```

Comment:
Add end of file (EOF)

Reply:
Te refieres a una linea limpia al final?

Reply:
dou you mean, the last cleean line at the end?